docs/development/release_management.rst

```diff
@@ -95,7 +93,7 @@ Pre-Release
       script, you will need to pass it the new version in the format
       ``<major>.<minor>.<patch>~rcN``::
 
-        securedrop/bin/dev-shell ../update_version.sh <major>.<minor>.<patch>~rcN
+        bin/dev-shell ../update_version.sh <major>.<minor>.<patch>~rcN
```

Comment:
Why remove the `securedrop` here? AFAICT `securedrop/bin/dev-shell` is still the correct invocation when you're in the repo root

Reply:
if you're in the repo root then the `../update_version.sh` is incorrect, and as far as i can tell, you have to be in the `securedrop` directory for this script to work. this is how it worked for me anyway.

Reply:
```securedrop/bin/dev-shell ls ..
1.8.0-rc1.tag	    SECURITY.md   codecov.yml	  mypy.ini
CODE_OF_CONDUCT.md  SOURCE_OFFER  devops	  securedrop
CONTRIBUTING.md     Vagrantfile   git		  securedrop-admin
LICENSE		    admin	  install_files   setup.py
MANIFEST.in	    ansible.cfg   journalist_gui  tails_files
Makefile	    build	  junit		  update_version.sh
README.md	    changelog.md  molecule

Reply:
Running `securedrop/bin/dev-shell ../update_version.sh 1.8.0~rc1` in the repo root has the expected behavior for me.

Reply:
oh i see it does work the way it was, but confusing when you look at the command because it looks like you are specifying a path from the parent directory that will not exist, i.e. `../update_version.sh` will take you the parent directory of the root, which does not contain `update_version.sh`. the reason it works is because that path is given to the container. that's the confusing part.

Reply:
That's my understanding as well. I don't have an opinion about the preferred invocation, but if we change the instructions we'll have to be explicit about changing into the `securedrop` subdirectory first.

Reply:
to avoid confusion, i'll update this to include cd'ing into securedrop -  i think that's the clearest the way this is currently written to work. any objections?